deploy/docker/seqr/Dockerfile

```diff
@@ -17,17 +17,27 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
-
-# install python
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    python-dev \
-    ca-certificates \
-    && wget https://bootstrap.pypa.io/get-pip.py \
-    && python get-pip.py \
-    && pip install --upgrade pip setuptools \
+#  install the packages necessary to build Python source:
+RUN apt update && apt install -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev \
+    libnss3-dev libssl-dev libreadline-dev libffi-dev libbz2-dev \
+    libncursesw5-dev libsqlite3-dev tk-dev libc6-dev ca-certificates \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+# downland and install Python from soruce code
+RUN PYTHON_VERSION="3.7.8" \
```

Comment:
Codacy found an issue: [Quote this to prevent word splitting.](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5815111)